package.json

```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "@thundra/core": "^2.2.3",
-    "all-contributors-cli": "^6.2.0",
+    "all-contributors-cli": "^6.7.0",
```

Comment:
I don’t think that bump is necessary, it will get installed automatically, as 6.7+ is within the range of ^6.2.0. You just need to your lock file

Reply:
Yeah, I bumped it by running `yarn add all-contributors-cli`, but I guess it could be done with `yarn upgrade all-contributors-cli`. I can rebase it out if you want, but I figure it doesn't hurt being explicit when bumping the lock.

Reply:
Yes, you are right. I missed the fact that this is an app and the lock file matters, not a library that is used somewhere :) Nevermind me, sorry for the noise

Reply:
No worries